doc/source/structures/misc/time.rst

```diff
@@ -54,7 +54,7 @@ Special variable TIME
 Using TIME to detect when the physics have been updated 'one tick'
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-kOS programs run however fast your computer's animation rate will allow, which can flow and change from one moment to the next depending on load. However, the physics of the universe get updated at a fixed rate according to your game settings (the default, as of KSP 0.25, is 25 physics updates per second)
+kOS programs run however fast your computer's animation rate will allow, which can flow and change from one moment to the next depending on load. However, the physics of the universe get updated at a fixed rate according to your game settings (the default is 25 physics updates per second)
```

Comment:
Agree, although this explanation is totally wrong now that I read it again (not your fault, - the original phrasing was wrong).  The game *tries to* keep the physics up to the game's physics setting, but does not guarantee it.  Under higher loads it will sacrifice other things (like the animation rate) to attempt to keep the physics rate up to at least N per second (default N=25), but it may not be able to.   When it is unable to, that's when you see the clock time in the upper-left of the screen turn yellow or red instead of green.